src/pot.sol

```diff
@@ -54,17 +54,17 @@ contract Pot is LibNote {
     }
 
     // --- Data ---
-    mapping (address => uint256) public pie;  // user Savings Dai
+    mapping (address => uint256) public pie;  // User Savings Dai [wad]
```

Comment:
So here's a case where I feel the original comment is not really accurate, because the quantity isn't Dai, it's "scaled dai", much like `art` is "scaled debt". So it would be fair, if a little cryptic, to call this "per-user scaled savings dai". More verbose but more clear would be "per-user savings dai divided by chi". I'm fine with either of those options.

Reply:
Agreed. Taking inspiration from `art // Normalised Debt`, what about `Normalised Savings Dai` ?

Reply:
Sure, let's go with that.